packages/seacas/libraries/ioss/src/exodus/Ioex_DatabaseIO.C

```diff
@@ -641,74 +641,85 @@ namespace Ioex {
     }
   }
 
-  void DatabaseIO::get_step_times_nl()
+  std::vector<double> DatabaseIO::internal_get_step_times_nl(bool setRegionTimeSteps)
   {
     bool                exists         = false;
     double              last_time      = DBL_MAX;
+    int                 tstepCount     = 0;
     std::vector<double> tsteps(0);
 
+    int& timestepCount = setRegionTimeSteps ? m_timestepCount : tstepCount;
```

Comment:
This is a little obscure since it is optionally setting the member variable.  A comment might be useful since it looks very strange why you have a reference here...

Reply:
Yeah, the reference is to make sure that no Region modifications occur based on the input flag but I can add a comment for that